internal/client/dispatch.go

```diff
@@ -88,10 +88,21 @@ dispatchLoop:
 
 		c.logDebug("Dispatching job %s", jobId)
 
-		// TODO implement retry
```

Comment:
This TODO is probably deprecated. Handle is a local function so wrapping it into a retry like the rest doesn't make sense.